app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/TranslationLabels.scala

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
If we are not using optFullNamePageLabel this class isn't necessary. In Sole trader we had

case class JourneyLabels(welsh: TranslationLabels)

but in incorporated entity we have

case class JourneyLabels (optWelshServiceName: String)

With hindsight I'm not sure TranslationLabels was ever really necessary, because you can define multipart JsPaths in object reads and writes. For example, JourneyLabels.